stories/topics/con-gcp-customize-monitoring-logging.adoc

```diff
@@ -3,9 +3,9 @@
 = Customizing monitoring and logging
 
 Metrics are provided by link:https://docs.ansible.com/automation-controller/latest/html/administration/metrics.html[Ansible], link:https://github.com/containers/prometheus-podman-exporter[Podman] and link:https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent[Google Ops Agent].
-Ansible metrics is only installed on {HubName}, the Google Ops Agent and Podman are also installed on {ControllerName}s.
+Ansible metrics is only installed on {HubName}, while the Google Ops Agent and Podman are installed on both {ControllerName} and {HubName} VM instances.
```

Comment:
The Google Ops Agent and Podman are installed on {ControllerName} and {HubName} VM instances, however Ansible metrics are only installed on {HubName} instances.